notebooks/tfco-fairlearn-experiment.ipynb

```diff
@@ -0,0 +1,414 @@
+{
```

Comment:
please remove :-)

Reply:
Oh this was intentional as I see now from the description. Hmm, I guess I have several reasons not to include it here at this point:

- new notebooks should go under examples/notebooks in .py format
- this introduces tensorflow as a dependency which we're definitely not going to add, so it would have to run in a separate pipeline with its own requirements.
- it's a separate contribution, best to keep the estimator copying as its own PR

Does that make sense?

Reply:
Yes, I think this makes sense. maybe later we can have one consolidated PR with all the notebooks. I will remove this from here.